Source/SwiftLintFramework/Rules/Lint/NoBlanketDisablesRule.swift

```diff
@@ -0,0 +1,135 @@
+import SwiftSyntax
+
+struct NoBlanketDisablesRule: ConfigurationProviderRule {
+    typealias ConfigurationType = NoBlanketDisablesConfiguration
+
+    var configuration = NoBlanketDisablesConfiguration()
+
+    init() {}
+
+    static let description = RuleDescription(
+        identifier: "no_blanket_disables",
+        name: "No Blanket Disables",
+        description: "swiftlint:disable commands should be re-enabled before the end of the file",
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("""
+            // swiftlint:disable unused_import
+            // swiftlint:enable unused_import
+            """),
+            Example("""
+            // swiftlint:disable unused_import unused_declaration
+            // swiftlint:enable unused_import
+            // swiftlint:enable unused_declaration
+            """),
+            Example("// swiftlint:disable:this unused_import"),
+            Example("// swiftlint:disable:next unused_import"),
+            Example("// swiftlint:disable:previous unused_import")
+        ],
+        triggeringExamples: [
```

Comment:
I did play with that before, but ran into some issues because the Command location was not where I expected it to be. I'll have another crack at it.